README.md

```diff
@@ -1 +1,39 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> This project is to build the structure/ wireframe for the leaderboard app.
+
+## Built With
+
+- html
+- css
+- javaScript
+- wepback
+
+## Author Okoro Priscilla Oyome
+
+👤 **Author**
+
+- GitHub: [@Prisca-tech](https://github.com/Prisca-tech)
+- Twitter: [@preateamie](https://twitter.com/preateamie)
+- LinkedIn: [PriscillaOkoro](https://www.linkedin.com/in/okoro-priscilla-oyome/)
+
+
+## 🤝 Contributing
```

Comment:
[OPTIONAL] You have an extra file in your project. `.vscode`. Kindly remove this file.